Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_005.SlackTokens.sarif

```diff
@@ -74,6 +74,7 @@
             }
           ],
           "fingerprints": {
+            "AssetFingerprint/v1": "",
```

Comment:
>AssetFingerprint [](start = 13, length = 16)

ok! now for slack if it validates we can provide account name details! open an issue to make this slack improvement. that improvement wouldn't actually update this test file, though, because you need to run dynamic validation to get the data.